test/com/jambit/DatabaseTest.java

```diff
@@ -0,0 +1,18 @@
+package com.jambit;
+
+import org.junit.jupiter.api.*;
+
+public class DatabaseTest {
+
+  private Database db = new Database();
```

Comment:
This test won't work if you can't reach the database. Use a mock instead.
https://www.tutorialspoint.com/mockito/mockito_junit_integration.htm

Reply:
would it be ok if we would use H2 for database connection testing ?